src/configs/ToastConfig.js

```diff
@@ -32,21 +34,19 @@ const styles = StyleSheet.create({
 const styleAsNative = StyleSheet.create({
   container: {
     backgroundColor: COLORS.black75,
-    borderRadius: 45,
-    paddingHorizontal: 30,
-    paddingVertical: 5,
-    width: 295,
-    minHeight: 42,
+    borderRadius: dimen.normalizeDimen(45),
+    paddingHorizontal: dimen.normalizeDimen(30),
+    paddingVertical: dimen.normalizeDimen(5),
+    width: dimen.normalizeDimen(295),
+    minHeight: dimen.normalizeDimen(42),
     justifyContent: 'center',
     alignItems: 'center'
   },
   text: {
     color: COLORS.white,
     textAlign: 'center',
-    fontFamily: 'Inter',
-    fontSize: 12,
-    fontStyle: 'normal',
-    fontWeight: '400',
-    lineHeight: 18
+    fontFamily: fonts.inter[400],
+    fontSize: normalizeFontSize(12),
+    lineHeight: normalizeFontSize(18)
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The style definitions for the container and text have been updated to use the `normalizeDimen` function from the `dimen` module for dimension values and the `fonts` object for the fontFamily property. The fontSize and lineHeight properties now use the `normalizeFontSize` function. These changes ensure that the styles are consistent and scalable across different screen sizes and devices.

```diff
-    borderRadius: 45,
-    paddingHorizontal: 30,
-    paddingVertical: 5,
-    width: 295,
-    minHeight: 42,
-    fontFamily: 'Inter',
-    fontSize: 12,
-    fontStyle: 'normal',
-    fontWeight: '400',
-    lineHeight: 18
+    borderRadius: dimen.normalizeDimen(45),
+    paddingHorizontal: dimen.normalizeDimen(30),
+    paddingVertical: dimen.normalizeDimen(5),
+    width: dimen.normalizeDimen(295),
+    minHeight: dimen.normalizeDimen(42),
+    fontFamily: fonts.inter[400],
+    fontSize: normalizeFontSize(12),
+    lineHeight: normalizeFontSize(18)
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->